decidim-admin/app/controllers/decidim/admin/application_controller.rb

```diff
@@ -17,6 +17,7 @@ class ApplicationController < ::DecidimController
       helper Decidim::Admin::IconLinkHelper
       helper Decidim::Admin::MenuHelper
       helper Decidim::Admin::ScopesHelper
+      helper Decidim::Admin::Paginable::PerPageHelper
```

Comment:
Why not declaring this helper inside the `Decidim::Admin::Paginable` concern itself?

Reply:
yes, why not!